### ISSUE TITLE
[Feature Request] Pinned messages

### ISSUE THREAD

**Issue by Lumenaut** [keybase/client#17556]
I'd love to see important messages pinned in chat channels, for easy reference.

This would be very useful for to-do lists, repeated information that has to be commented various times, team/project updates, important warnings, help request, etc.

Ideally, the pinned messages should be allowed to be made by "Admins" or "Writers" to avoid abuse.

**Comment by ChrisRenfrow** [comment]
@Lumenaut +1 on this as this is a major feature a lot of slack users will be looking for when making the transition to Keybase

**Comment by maxtaco** [comment]
In progress

**Comment by ctsstc** [comment]
Seems there is a pinned message, but looks like you can only have one?

**Comment by ChrisRenfrow** [comment]
@ctsstc Correct, you can have one pinned message currently. But we often have several things we'd like to be collected together for new members to refer to. 

Currently our solution is to have one "mega-pin" in which one message contains all of the content we'd like pinned in a channel. It's not awful to do it this way but I'd prefer to not have to dig up "ancient messages :moyai:" just to add new pinned information.

**Comment by ajmeese7** [comment]
I would very much like to have multiple messages pinned! That's the only feature lacking from Discord that I've really missed since my transition to Keybase, I would love to see it implemented
